NomadAudio/src/Win32/WASAPISharedDriver.cpp

```
@@ -713,6 +722,12 @@ void WASAPISharedDriver::audioThreadProc() {
         // Release buffer
         hr = reinterpret_cast<IAudioRenderClient*>(m_renderClient)->ReleaseBuffer(availableFrames, 0);
         if (FAILED(hr)) {
+             if (hr == AUDCLNT_E_DEVICE_INVALIDATED || 
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)